src/main/java/org/folio/service/invoice/PoLineInvoiceLineHolderBuilder.java

```diff
@@ -53,7 +53,7 @@ private CompletableFuture<List<InvoiceLine>> validateAndRetrievePaidInvoiceLines
       String poLineFundId = optionalFundDistribution.get().getFundId();
       return getCurrentFiscalYearInvoiceLines(poLineFundId, invoiceLines, requestContext)
         .thenApply(currentYearInvoiceLines -> {
-          validateInvoiceLinesStatus(currentYearInvoiceLines);
+//          validateInvoiceLinesStatus(currentYearInvoiceLines);
```

Comment:
This change will be uncommented when Flow #2 for Edit Fund Distribution feature will be implemented